pages/form-builder/support/[[...supportType]].tsx

```diff
@@ -487,7 +486,7 @@ export default function Contactus() {
           <h1 ref={headingSuccessRef}>{t("requestSuccess.title")}</h1>
           <p className="mb-16 mt-[-2rem] font-bold">{t("requestSuccess.weWillRespond")}</p>
           <div className="mb-16">
-            <StyledLink href={`/myforms`} className="gc-button--blue">
+            <StyledLink href={`/myforms`} theme="primaryButton">
```

Comment:
Should be able to use <LinkButton.Primary here

Reply:
Got it.  So StyledLink is so out and LinkButton.Primary is so hot right now :D